src/component/addedittimeline/index.tsx

```diff
@@ -48,7 +49,6 @@ function AddEditTimeline(props: AddEditTimelineProps) {
   const token = useGetToken();
 
   const isEdit = timelineSlug ? "isEdit" : undefined;
-  const [visibility] = useState("private");
```

Comment:
Did you have plans for this hook? Not currently used.

Reply:
Yes, in addedittimeline we plan to add a permissions radio which assigns timeline visibility.

You're welcome to do whatever you think makes sense.
![shared_-_Plasmic](https://user-images.githubusercontent.com/1753324/118380439-938ac880-b596-11eb-984f-20c137708634.jpg)

Reply:
Thanks for the info. I'll bring it back when we're ready for that feature since it'll require more state management probably.